docs/en/architecture.md

```diff
@@ -0,0 +1,22 @@
+Deploying this solution with the default parameters builds the following environment in the AWS Cloud.
+
+![architecture](./images/arch.png)
+*Figure 1: AI Video Super Resolution architecture*
+
+This solution deploys the Amazon CloudFormation template in your Amazon Cloud Technology account and completes the following settings.
```

Comment:
what is `Amazon Cloud Technology`? it has multiple occurrences

Reply:
should be AWS CloudFormation,

https://aws.amazon.com/cloudformation/

Reply:
"This solution will deploy following resources into your AWS account" should be more accurate, also not AWS can be present in github